packages/mocks/package.json

```diff
@@ -18,10 +19,12 @@
     "express": "^4.18.2",
     "express-http-proxy": "^1.6.3",
     "minimist": "^1.2.8",
-    "msw": "^1.2.1",
-    "process": "^0.11.10"
+    "msw": "^1.2.1"
   },
   "devDependencies": {
     "ts-node": "^10.9.1"
+  },
+  "msw": {
+    "workerDirectory": "static"
```

Comment:
In #365  the folder is named `generated`. IMHO this is more explicit then `static`  which may contain hand-written code.

Reply:
Changed.